src/controllers/cache.ts

```diff
@@ -79,6 +79,7 @@ function forEach(
   return cache[table].forEach(callback);
 }
 
+// deno-lint-ignore prefer-const
 export let cacheHandlers = {
```

Comment:
```suggestion
export const cacheHandlers = {
```

Reply:
We can just make this a const no?

Reply:
@Skillz4Killz in case, the users need to modify the value.